<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableInfoDialog</class>
 <widget class="QDialog" name="TableInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Table name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>&lt;name&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="infosTab">
      <attribute name="title">
       <string>General informations</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="columnTab">
      <attribute name="title">
       <string>Columns</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="addColumnButton">
         <property name="text">
          <string>&amp;Add</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/img/insert_table_col.png</normaloff>:/img/insert_table_col.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="removeColumnButton">
         <property name="text">
          <string>&amp;Remove</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/img/delete_table_row.png</normaloff>:/img/delete_table_row.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QTableWidget" name="columnView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="prevTableButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Previous</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/img/previous.png</normaloff>:/img/previous.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="nextTableButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Ne&amp;xt</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/img/next.png</normaloff>:/img/next.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TableInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
